api/server/sdk/server_interceptors.go

```diff
@@ -141,6 +155,12 @@ func (s *sdkGrpcServer) authorizationServerInterceptor(
 		"method":   info.FullMethod,
 	})
 
+	// If a public create call is attempted, but public volume creation is disabled, then deny
+	if auth.IsPublic(ctx) && s.config.Security.PublicVolumeCreationDisabled &&
```

Comment:
We can remove this and let the RBAC system take care of it

Reply:
This will be refactored once we have support for negation in the RBAC system.